src/common/hook/usePublishQueue/usePublishQueue.tsx

```diff
@@ -41,15 +43,25 @@ export const usePublishQueue = (
     };
   });
 
+  const pendingPublishDocuments = pendingJobIndex.reduce(
+    (acc, curr) => [...acc, ...jobs[curr].documents],
+    [] as WrappedDocument[]
+  );
+
   const publish = async (): Promise<void> => {
     try {
       // Cannot use setCompletedJobIndex here as async update does not with the promise race
       const completedJobs: number[] = [];
       const failedJobs: FailedJob[] = [];
+      // let pendingJobs: number[] = [];
+
       setPublishState("INITIALIZED");
       const nonce = await config.wallet.getTransactionCount();
       const publishingJobs = await getPublishingJobs(formEntries, config, nonce);
       setJobs(publishingJobs);
+      // pendingJobs = publishingJobs.map((job, index) => index);
+      const pendingJobs = new Set(publishingJobs.map((job, index) => index));
```

Comment:
Does eslint complain about unused `job` variable?

```suggestion
      const pendingJobs = new Set(publishingJobs.map((, index) => index));
```

Reply:
no leh nvr complain. ok i'll update it